### PR TITLE
Micro-Reddit Setup

### PR DESCRIPTION
**Pull request by carlosveigadev**
## What this PR for: 
- Create generate rails app;
- Add main files;
- Create User model and add validations to it;
- Create Posts model and add validations to it;
- Create Comments model and add validations to it;
- Migrate DB;
- Update Readme files;
- Fix rubocop offenses.

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+  has_many :posts
+
```
Comment (addod19) [inline app/models/user.rb:4]: Hi, we are missing the relationship between `user` and `comments`, let us add `has_many :comments`

File: app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+
+  validates :user_id, presence: true
```
Comment (addod19) [inline app/models/post.rb:5]: Hi, we are missing validations for `body` and `title` of our post. Let us add them

File: app/models/comment.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
```
Comment (addod19) [inline app/models/comment.rb:4]: Hi, we are missing validations for our comment `body`. Let us validate it

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi, @wrakc and @sanadwj, great job with this project :+1: The project works as expected, just a little change to effect. Kindly check the comments below to address the issue: :smile: 
- Kindly implement a test for this project :+1: 
- Kindly implement the issues with the association and validations outlined

Once this is fixed, kindly submit another code review :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by carlosveigadev** [comment]
Hi @addod19, thanks for the review.
We improved the code as requested.

**Review by uche-inyama — APPROVED** [review]
## Approved

Hi team :wave: 

Good job implementing the changes.

Remove validations for ids in your models: post and comment.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry Inyama ```

**Pull request merged**